From d3d34949d54c1e4e7777a8cbcd7f0d3d78c8448a Mon Sep 17 00:00:00 2001
From: Brad Geltz <brad.geltz@intel.com>
Date: Wed, 7 Oct 2020 11:20:39 -0700
Subject: [PATCH 1/2] Fix whitespace issues
#
#  Copyright (c) 2015 - 2021, Intel Corporation
#
#  Redistribution and use in source and binary forms, with or without
#  modification, are permitted provided that the following conditions
#  are met:
#
#      * Redistributions of source code must retain the above copyright
#        notice, this list of conditions and the following disclaimer.
#
#      * Redistributions in binary form must reproduce the above copyright
#        notice, this list of conditions and the following disclaimer in
#        the documentation and/or other materials provided with the
#        distribution.
#
#      * Neither the name of Intel Corporation nor the names of its
#        contributors may be used to endorse or promote products derived
#        from this software without specific prior written permission.
#
#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY LOG OF THE USE
#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
#

- Add build files to .gitignore

Signed-off-by: Brad Geltz <brad.geltz@intel.com>
---
 .gitignore                | 3 +++
 setup/Make.IMPI_IOMP_AVX2 | 6 +++---
 setup/Make.IMPI_IOMP_SKX  | 6 +++---
 3 files changed, 9 insertions(+), 6 deletions(-)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..14ba1b2
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,3 @@
+*.o
+bin/
+Makefile
diff --git a/setup/Make.IMPI_IOMP_AVX2 b/setup/Make.IMPI_IOMP_AVX2
index e7e42ca..38b0f73 100644
--- a/setup/Make.IMPI_IOMP_AVX2
+++ b/setup/Make.IMPI_IOMP_AVX2
@@ -87,9 +87,9 @@ BINdir       = $(TOPdir)/bin
 # header files,  MPlib  is defined  to be the name of  the library to be
 # used. The variable MPdir is only used for defining MPinc and MPlib.
 #
-MPdir        = 
-MPinc        = 
-MPlib        = 
+MPdir        =
+MPinc        =
+MPlib        =
 #
 #
 # ----------------------------------------------------------------------
diff --git a/setup/Make.IMPI_IOMP_SKX b/setup/Make.IMPI_IOMP_SKX
index 3213ef7..9325d71 100644
--- a/setup/Make.IMPI_IOMP_SKX
+++ b/setup/Make.IMPI_IOMP_SKX
@@ -87,9 +87,9 @@ BINdir       = $(TOPdir)/bin
 # header files,  MPlib  is defined  to be the name of  the library to be
 # used. The variable MPdir is only used for defining MPinc and MPlib.
 #
-MPdir        = 
-MPinc        = 
-MPlib        = 
+MPdir        =
+MPinc        =
+MPlib        =
 #
 #
 # ----------------------------------------------------------------------
-- 
2.23.0

